src/ConfigProvider.php

```diff
@@ -27,11 +27,6 @@ public function getDependencies(): array
         return [
             'aliases'   => [
                 TemplateRendererInterface::class => LaminasViewRenderer::class,
-
-                // Legacy Zend Framework aliases
-                'Zend\Expressive\Template\TemplateRendererInterface' => TemplateRendererInterface::class,
-                'Zend\View\HelperPluginManager'                      => HelperPluginManager::class,
-                'Zend\Expressive\ZendView\ZendViewRenderer'          => LaminasViewRenderer::class,
```

Comment:
These aliases have nothing to do with the components being installed: they are mere service names, so conflicting with `"zendframework/zend-view": "*"` doesn't allow us to drop them.

Re-target `3.0.x` and we're good :+1: